vgc/pipeline.py

```diff
@@ -1,50 +1,84 @@
-import logging
-from .log import log_init
+"""
+Class Pipeline which initializes modules and works
+as a middle hand for communication.
+"""
+from .log import logger
 from .camera_filter import CameraFilter
 from .view_controller_fast import ViewController
 from .io.input_adapter import InputAdapter
+from .autopilot_adapter import Vehicle
 from .config import CONFIG
 
+# pylint: disable=logging-fstring-interpolation
+# Don't really know what this is, but the part of the code
+# where pylint is angry works perfectly.
+
+# pylint: disable=too-many-arguments
+# 6 arguments are needed.
+
 class Pipeline:
     """"
     Cross thread messaging pipeline.
     Threads communicates by calling pipeline's
     class functions
 
-    __init__(self):
```

Comment:
Should there really be a description of how the class functions are used here? The same description is writen under all functions